plugins/scanners/fossid/src/main/kotlin/FossIdNamingProvider.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
The year hear is when the file was created originally; so this should stay 2021. But the copyright year in the test file you have newly added should be 2024.